src/Resources/config/doctrine/BaseMedia.orm.xml

```diff
@@ -16,7 +16,7 @@
         <field name="height" column="height" type="integer" nullable="true"/>
         <field name="length" column="length" type="decimal" nullable="true" precision="10"/>
         <field name="contentType" column="content_type" type="string" nullable="true" length="255"/>
-        <field name="size" column="content_size" type="integer" nullable="true"/>
+        <field name="size" column="content_size" type="integer" length="5" nullable="true"/>
```

Comment:
Updating the existing config would trigger a migration for all existing user no ? Could it be considered as a BC break ?

Reply:
Yes it will, but we are widening it, so should be ok, also if you don't execute a new migration the code would still work like before.

Reply:
We had some discussion about such schema changes and it was unclear about if it's okay or not to introduce a migration for users https://github.com/sonata-project/SonataUserBundle/pull/1677

I assume that changing array to json is a BC break because the migration is required but maybe changing int to bigint is ok if the code still work without the migration (?).

@core23 @jordisala1991 WDYT ?

Reply:
Ping @dmaicher

Reply:
Ping @dmaicher

Reply:
Out of curiosity: Why not use `bigint`?
```suggestion
        <field name="size" column="content_size" type="bigint" nullable="true"/>
```

Reply:
Does this work with Postgres?

Reply:
Hm I guess so :thinking: 

See https://www.doctrine-project.org/projects/doctrine-dbal/en/4.2/reference/types.html#bigint

Not sure which DBAL version we support here and if that makes a difference though

Reply:
Ok, already supported in DBAL 3, so lets use that